cookbooks/fb_fstab/spec/public_spec.rb

```diff
@@ -58,7 +57,6 @@
 proc                    /proc                   proc    defaults        0 0
 tmpfs /dev/shm tmpfs defaults,size=4G 0 0
 EOF
-  # rubocop:enable LineLength
 
   context 'generate_base_fstab' do
     it 'should not regenerate base fstab' do
```

Comment:
Please update this test, line 65 to reflect the change from `File.size` to `File.size?` in the method:
```
--- a/chef/cookbooks/core/fb_fstab/spec/public_spec.rb
+++ b/chef/cookbooks/core/fb_fstab/spec/public_spec.rb
@@ -62,7 +62,7 @@
     it 'should not regenerate base fstab' do
       File.should_receive(:exist?).with(FB::Fstab::BASE_FILENAME).
         and_return(true)
-      File.should_receive(:size).with(FB::Fstab::BASE_FILENAME).
+      File.should_receive(:size?).with(FB::Fstab::BASE_FILENAME).
         and_return(12)
       File.should_not_receive(:open)
       FB::Fstab.generate_base_fstab
```